msal/src/Microsoft.Identity.Client/Features/ConfidentialClient/JsonWebToken.cs

```diff
@@ -86,10 +87,10 @@ public JsonWebToken(string clientId, string audience)
             };
         }
 
-        public string Sign(ClientAssertionCertificate credential)
+        public string Sign(ClientAssertionCertificate credential, bool sendX5C)
```

Comment:
>sendX5C [](start = 71, length = 7)

as Petro suggested. stick to one name all the way through. Ideally the one agreed to in the public API with the PM